lib/Wikia/src/Service/User/Permissions/PermissionsService.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+interface PermissionsService {
+
+	/**
+	 * Gets the object containing definitions of groups and permissions in the system
+	 * @return PermissionsDefinition
+	 */
+	public function getDefinitions();
+
+	/**
+	 * @param \User $user
+	 * @return string[] Get list of global and local groups to which the given user has been explicitly assigned to.
+	 */
+	public function getExplicitUserGroups( \User $user );
```

Comment:
all the methods in this class accept a `User` object and operate on user, can we get rid of all the `User`'s in the method names?